thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/alert/commons/AnomalyFeedConfig.java

```diff
@@ -19,7 +19,7 @@
 
 package org.apache.pinot.thirdeye.alert.commons;
 
-import org.apache.pinot.thirdeye.datalayer.util.StringUtils;
+import org.apache.pinot.thirdeye.datalayer.util.ThirdEyeStringUtils;
```

Comment:
Any reason for this name change?
"thirdeye" already in the package name.

Reply:
There is a standard `StringUtils` utility class in apache commons which is widely used. Having two such utility classes requires us to distinguish between them by referring to their complete package name.

Reply:
Sounds good. Thanks.